source/includes/api/tables/progress-response.rst

```diff
@@ -129,3 +129,7 @@
        - When ``mongosync`` starts up, this field returns ``null`` until
          ``mongosync`` identifies the coordinator.
 
+   * - ``success``
+     - boolean
+     - Status of the ``progress`` command. Value is ``true`` if the 
+       command succeeds and ``false`` if the command fails. 
```

Comment:
This table item is a little misleading. This table is introduced by the sentence
> If mongosync successfully gets the status of the sync process, all response fields are wrapped in a top-level `progress` object with the following fields:

The `success` key is a top-level field, not part of the `progress` object. That is: a successful response looks like this:
```javascript
{
  success: true,
  progress: {
    // All the other fields as in the table
  }
}
```

So, this description is fine, but it's a top-level field and not part of the `progress` object.